### ISSUE TITLE
SDL1 - color palette changes

### ISSUE THREAD

**Issue by emendelson** [joncampbell123/dosbox-x#1459]
This problem occurs with SDL1 builds of current code (in Windows; I haven't tested other builds): in certain situations, the color palette changes and can't be restored without restarting DOSBox-X. The problem can be seen this way.

1. Download and extract the WordPerfect Office 3.1 package (released freely by Novell many years ago). I've made a ZIP file here:

http://www.columbia.edu/~em36/WPShell.zip

2. In DOSBox-X, mount the directory with the programs as a drive. Change to that drive and run Shell.exe. The Shell menu will have a blue background.

3. In the Shell menu, press A to open the appointment calendar. The background will remain blue. After the calendar appears, press F7 to exit (F7 is the WordPerfect exit key).

4. The Shell menu will appear again - but the background will now be pale green instead of blue.

In SDL2 builds, the background will remain blue instead of turning green.

Almost no one cares about the WordPerfect Shell, but this may indicate an underlying problem that might be worth looking at.

**Comment by Wengier** [comment]
@emendelson Hmm, I tried the WPShell files (downloaded from your link) in DOSBox-X 0.83.0 and latest commit (SDL1 version), but the WPOffice background remains blue even after I press F7 to exit from the Appointment Calendar. I am using Windows 10 64-bit, tested with the default dosbox.conf file and my customized one. Perhaps you are using some modified dosbox.conf file which will result in this issue? Or maybe you are using some slightly different environment?

**Comment by emendelson** [comment]
@Wengier - This is very strange. I repeatedly got that green background earlier today, and now the background stays blue (although there is a quick flash of green when the Shell starts up). I think something must have gone wrong in my Windows video settings, and it got fixed after I restarted the machine. 

Unless I find what caused this, there's no point in spending any more time on it. I'll close this, with apologies for wasting bandwidth.
